### PR TITLE
Feature/topbar documentation

### PR DESCRIPTION
**Pull request by eskilsen**
Update topbar documentation page
Also update topbar div to have a div wrapped around contents for making the yellow line always be edge to edge. 
## How Has This Been Tested?

Ran all tests, updated snapshots. 
## Screenshots (if appropriate):
Before: 
image
After: 
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Review instructions
[Review instructions](../REVIEW_INSTRUCTIONS.md)

### CODE REVIEW

File: src/App/ComponentsDocumentation/components/Topbar/index.js [hunk 1]
```diff
@@ -119,14 +107,61 @@ const TopbarPNG = () => (
...
-        <h2 id="javascript-methods">JavaScript Methods</h2>
-        <JavascriptDocs componentName="topbar" open close />
-    </>
+const DeveloperDocumentation = () => (
+    <section>
+        <h2 id="developer-documentation">Developer documentation</h2>
+
+        <section>
+            <h3>Align with page content</h3>
+            <p>The content within the desktop topbar, <CodeTags type="secondary" code=".topbar-xl-wide" />, is by default fluid to the screen size. If you want to restrict the width of the content within the topbar to align with your page container, simply add a <CodeTags type="secondary" code=".max-width" />     to the topbar <CodeTags type="secondary" code=".nav-container" /> to match it.</p> {/* TODO: Whats this ... ? */}
```
Comment (eskilsen) [inline src/App/ComponentsDocumentation/components/Topbar/index.js:119]: ?

File: src/App/ComponentsDocumentation/components/Topbar/index.js [hunk 2]
```diff
@@ -119,14 +107,61 @@ const TopbarPNG = () => (
         <ComponentPreview language="html" showCasePanel codeFigure>
             <TopbarComponent id="demo-topbar-1-png" topbarContent={menu} wide="xl" logout png/>
         </ComponentPreview>
-    </>
+    </section>
 );
 
-const JavascriptMethods = () => (
```
Comment (eskilsen) [inline src/App/ComponentsDocumentation/components/Topbar/index.js:125]: Remove

File: src/App/ComponentsDocumentation/components/Topbar/index.js [hunk 3]
```diff
@@ -32,30 +34,15 @@ const menu = {
...
-            Note that scrolling is locked when the menu is displayed.
-        </p>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <TopbarComponent topbarContent={menu} logout id="demo-topbar" />
-        </ComponentPreview>
-    </>
+    </section>
 );
 
 const TopbarWide = () => (
```
Comment (eskilsen) [inline src/App/ComponentsDocumentation/components/Topbar/index.js:48]: Remove

**Review by hmosvoll — APPROVED** [review]
LGTM

**Pull request merged**